node/setup.go

```diff
@@ -259,6 +264,8 @@ func createMempoolAndMempoolReactor(
 		// Strictly speaking, there's no need to have a `mempl.NopMempoolReactor`, but
 		// adding it leads to a cleaner code.
 		return &mempl.NopMempool{}, mempl.NewNopMempoolReactor()
+	case cfg.MempoolTypeCAT:
```

Comment:
is cat not implemented?

also a separate question, we no longer refer to mempools as v0, v1, v2?

Reply:
cat hasnt been ported yet, i was waiting on getting this work completed. not sure why its not v0,v1,v2 anymore this was adopted by comet.